docs/serving/services/deployment.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Deployment Configuration"
+linkTitle: "Deployment Configuration"
+weight: 03
+type: "docs"
+---
+
+The `config-deployment` ConfigMap is located in the `knative-serving` namespace.
+This ConfigMap, known as the Deployment ConfigMap, contains settings that determine how Kubernetes `Deployment` resources, that back Knative services, are configured.
+
+## Accessing the Deployment ConfigMap 
+
+To view the current Deployment ConfigMap:
+
+```bash
+kubectl get configmap -n knative-serving config-deployment -oyaml
+```
+
+## Progress Deadline Seconds
+
+ProgressDeadline setting specifies the value of the `.spec.progressDeadlineSeconds` field on the underlying Kubernetes deployment for the Knative Revision.
+By default this value is set to 120 seconds.
+The value is expressed as a Go `time.Duration` string representation, but must be rounded to a second precision.
+Please see [Kubernetes documentation](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#progress-deadline-seconds) for details about this setting.
+
+In Knative the Autoscaler component gives up and scales the revision to 0 if the initial scale cannot be achieved within approximately this time span. If this happens the revision is considered to be failed.
+In the following cases it might make sense to change this setting to a higher value:
+
+- it takes a lot of time to pull the Service image (e.g. they are very big) 
+- it takes a lot of time to make the service ready (e.g. priming initial cache state)
+- the cluster is relying on cluster autoscaler to allocate resources for new pods.
```

Comment:
I'd delete this and move it higher in the doc as per suggestion comment above

Reply:
Commited the other change.